src/components/common/datagrid.js

```diff
@@ -90,9 +90,13 @@ const StyledCell = styled.div`
   height: 100%;
```

Comment:
when `span` is in `StyledCell` text-overflow css needs to be applied to `span`, instead of the parent `div`
```
  span {
    overflow: scroll;
    text-overflow: ellipsis;
    white-space: pre-wrap;
    word-wrap: break-word;
    width: 100%;
  }
```

Reply:
Also maybe add left padding to the first cell
```
const DataGridWrapper = styled.div`
   .....other css
  .ReactVirtualized__Grid {
    .column-0 .cell{
      padding-left: 24px;
    }
  }
```